lib/geoengineer/resources/aws_dynamodb_table.rb

```diff
@@ -12,6 +12,18 @@ def support_tags?
     false
   end
 
+  def to_terraform_state
+    tfstate = super
+    return tfstate unless self.ttl
+
+    tfstate[:primary][:attributes] = {
+      "ttl.#" => "1",
+      "ttl.1794544261.attribute_name": ttl.attribute_name.to_s,
+      "ttl.1794544261.enabled": ttl.enabled.to_s
```

Comment:
I would add a comment saying why this number was chosen (I understand it's "random", but this may not be obvious to other readers of this code).